aipl/main.py

```diff
@@ -4,15 +4,22 @@
 
 from aipl import AIPL, Table, UserAbort
 
-
-def main():
+def parse_args(args):
     parser = argparse.ArgumentParser(description='AIPL interpreter')
-    parser.add_argument('--visidata', '--vd', action='store_true', help='open VisiData with input before each step')
     parser.add_argument('--debug', '-d', action='store_true', help='abort on exception')
-    parser.add_argument('--single-step', '-x', action='store_true', help='breakpoint() before each step')
+    parser.add_argument('--step', action='store', default='', help='call aipl.step_<func>(cmd, input) before each step')
+    parser.add_argument('--step-breakpoint', '-x', action='store_const', dest='step', const='breakpoint', help='breakpoint() before each step')
+    parser.add_argument('--step-rich', '-v', action='store_const', dest='step', const='rich', help='output rich table before each step')
+    parser.add_argument('--step-vd', '--vd', action='store_const', dest='step', const='vd', help='open VisiData with input before each step')
+    parser.add_argument('--dry-run', '-r', action='store_true', help='do not execute @expensive operations')
```

Comment:
Could we make the short version of this `-n`? It feels like the common abbreviation to me.